core/unit_test/view/TestViewCustomizationAccessorFromMapping.hpp

```diff
@@ -216,6 +216,21 @@ TEST(TEST_CATEGORY, view_customization_accessor_from_mapping) {
 
 // This tests the ability to pass in the accessor arg
 // as an additional integral argument to constructor and shmem_size
+
+template <class ViewType, size_t... Idx>
+bool test_device_side_ctor(ViewType a, std::index_sequence<Idx...>) {
+  int error = 0;
+  Kokkos::parallel_reduce(
+      Kokkos::RangePolicy<TEST_EXECSPACE>(0, 1),
+      KOKKOS_LAMBDA(int, int& errors) {
+        ViewType a2(a.data(), a.extent(Idx)..., a.accessor().size);
+        if (a2.accessor().size != a.accessor().size) errors++;
+        if (a2.accessor().stride != a.accessor().stride) errors++;
```

Comment:
Is the size not going to be zero for all ranks >= 1 since your index sequence starts at 0

Reply:
I don't know what you mean? The accessor().size is always 5, on top of that the extents are non-zero too - I use the index sequence to index not directly as extents.